docs/source/examples/e2e_flow.rst

```diff
@@ -0,0 +1,438 @@
+.. _e2e_flow:
+
+==================================
+End-to-End Workflow with TorchTune
+==================================
+
+In this tutorial, we'll walk through an end-to-end example of how you can fine-tune,
+evaluate, optionally quantize and then run generation with your favorite LLM using
+TorchTune. We'll also go over how you can use some of your favorite tools and libraries
```

Comment:
nit: You say "favorite" twice in two sentences. Opt for a different word.

Reply:
s/favorite/popular ?